packages/app-headless-cms/src/admin/components/ContentModelForm/ContentModelFormRender/fields/DateTimeWithoutTimezone.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Input from "./Input";
+import { Grid, Cell } from "@webiny/ui/Grid";
+
+const DEFAULT_TIME = "00:00:00"
+const DEFAULT_DATE = "";
+
+const DateTimeWithoutTimezone = (props) => {
+  // "2020-05-18 09:00:00"
+  const [date, setDate] = React.useState("")
+  const [time, setTime] = React.useState("")
+  React.useEffect(() => {
+    if (props.bind.value === null) {
+      setDate(DEFAULT_TIME)
+      setTime(DEFAULT_DATE)
+      return
+    }
+    const [isoDate, fullTime] = props.bind.value.split(' ')
+
+    const formattedDate = isoDate
+    const formattedTime = fullTime
+
+    // Set previously saved values
+    if (date !== formattedDate) {
+      setDate(formattedDate)
+    }
+    if (time !== formattedTime) {
+      setTime(formattedTime)
+    }
+
+  }, [props.bind.value])
+
+  return (
+    <Grid>
```

Comment:
Does this look OK in the actual UI? I mean, adding the grid here and cell doesn't misalign the input with other inputs? I think the Grid adds some padding into the mix, doesn't it?

Reply:
A screenshot would be nice.

Reply:
Yeah, It should misalign/act for the reasons you mentioned above.
But, for some reason it's looking aligned :thinking: 
![image](https://user-images.githubusercontent.com/13612227/81829708-d1ef4100-9558-11ea-82a8-22d486e0ce1b.png)
![image](https://user-images.githubusercontent.com/13612227/81829797-eb908880-9558-11ea-9f54-b038e084f0fc.png)

Reply:
OK great :)